lib/rules/invocable-blacklist.js

```diff
@@ -13,7 +13,14 @@ module.exports = class InvocableBlacklist extends Rule {
         }
         break;
       case 'object':
-        if (Array.isArray(config) && config.length > 0) {
+        if (
+          Array.isArray(config) &&
+          config.length > 0 &&
+          config.every(
+            // Ensure names are passed as kebab-case strings.
+            (name) => typeof name === 'string' && name.length > 0 && dasherize(name) === name
```

Comment:
Hmm, can we just convert it for them? Why make it error (or did I misunderstand)?

Reply:
Are you suggesting allowing a component name to be passed in either of these two formats?

* `some-scope/component-name` (kebab-case)
* `SomeScope::ComponentName` (angle bracket style)

If you prefer, I could switch to that as the requirement. My original concern was that allowing multiple formats could create inconsistency or confusion (some people pass it one way, other people pass it another way).

Reply:
Yeah, I definitely see what you mean here. My main concern is that _for the most part_ folks will only think about the angle bracket invocation style (at some point in the future), and its going to end up being error prone in configuration.

I'd like to think about that a bit more, there is definitely nothing wrong with the implementation here. We can follow up and change things to allow either formats later if it turns out to be confusing.

Reply:
Sounds good. I'll leave as is for now, but definitely open to adjusting.